var/spack/repos/builtin/packages/serf/package.py

```diff
@@ -19,13 +19,13 @@ class Serf(SConsPackage):
     variant('debug', default=False,
             description='Enable debugging info and strict compile warnings')
 
-    depends_on('scons@2.3.0:', type='build')
-
     depends_on('apr')
     depends_on('apr-util')
     depends_on('openssl')
-    depends_on('zlib')
+    depends_on('python', type='build')
```

Comment:
```suggestion
    depends_on('python+pythoncmd', type='build')
```
Might want to add this until #25408 gets resolved. Or you could add a `patch()` method to fix it in this PR.

Reply:
Thanks for your proposal; I did not know this variant before. It was added.